docs/src/guide/installation.md

```diff
@@ -54,17 +99,6 @@ For Ubuntu 20.04LTS, these commands install the basic build system:
 	If you get errors about missing `pyelftools`, check that you're really
 	running SCons with Python 3!
```

Comment:
Is this even necessary at all anymore? Scons 4 works only with Python3, so it won't be choosing the next-best interpreter at runtime: https://scons.org/scons-400-is-available.html

I removed this from the macOS guide for this reason.

Reply:
But Ubuntu provides [Scons 3](https://packages.ubuntu.com/focal/scons).

Ubuntu 20.04 has Python3 and therefore no problem anyway.
I would keep the warning for now to give a hint to users of Ubuntu ≤ 19.10.

Reply:
Ok, good point. I'll squash and merge then.